src/app/main.ml

```diff
@@ -150,6 +151,7 @@ let begin_program
         Usbmux.Logging.(
           let relay_with =
             R.make_tunnels
+              ~bind_host:bind_host
```

Comment:
Look at this gist 

https://gist.github.com/fxfactorial/21b78bb2cae253a1b798b497ae35ae67